src/handlers/viewSubmitInteraction.ts

```diff
@@ -0,0 +1,76 @@
+import { IModify, IRead } from '@rocket.chat/apps-engine/definition/accessors';
+import { UIKitViewSubmitInteractionContext } from '@rocket.chat/apps-engine/definition/uikit';
+
+import { TodoistApp } from '../../TodoistApp';
+import { ModalsEnum } from '../enums/Modals';
+import { createTask } from '../lib/create/createTask';
+import { deleteComment, deleteLabel, deleteSection, deleteTask } from '../lib/delete/deleteItem';
+
+export class ExecuteViewSubmitHandler {
```

Comment:
hey, rename the file to match the class name

Reply:
Also, I think it should be implementing an interface, it is just working due to duck typing